packages/form-js-viewer/test/spec/render/components/form-fields/Textarea.spec.js

```diff
@@ -0,0 +1,255 @@
+import {
+  fireEvent,
+  render
+} from '@testing-library/preact/pure';
+
+import Textarea from '../../../../../src/render/components/form-fields/Textarea';
+
+import {
+  createFormContainer,
+  expectNoViolations
+} from '../../../../TestHelper';
+
+import { WithFormContext } from './helper';
+
+const spy = sinon.spy;
+
+let container;
+
+
+describe('Textarea', function() {
```

Comment:
Don't we have any test coverage for the auto resize behavior?

Reply:
There now is.